save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ExecutionView.kt

```diff
@@ -199,6 +199,20 @@ class ExecutionView : AbstractView<ExecutionProps, ExecutionState>(false) {
                         }
                     }
                 }
+                column(id = "missing", header = "Missing") {
+                    buildElement {
+                        td {
+                            +"${it.value.missing}"
```

Comment:
What if it's null? Also, it will always be null for fix-plugin - it needs to be handled correctly.

Reply:
Yes. Instead of a null, we will output an empty string